xml/System.Runtime.Serialization/SerializationInfo.xml

```diff
@@ -1043,7 +1043,7 @@ End Sub
       </Parameters>
       <Docs>
         <param name="name">The name to associate with the value, so it can be deserialized later.</param>
-        <param name="value">The <see langword="UInt64" /> value to serialize.</param>
+        <param name="value">The <see cref="T:System.UInt64" /> value to serialize.</param>
```

Comment:
If I interpret the [style guidelines](https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors) correctly, the type just shouldn't even be here.

```suggestion
        <param name="value">The value to serialize.</param>
```

But I agree the change as proposed is an improvement.

Reply:
Yep, that's a good suggestion @bartonjs.